README_BEAGLEBONE_BLACK.md

```diff
@@ -0,0 +1,162 @@
+Building SuperCollider on BeagleBone Black
+==========================================
+
+This is for compiling and running sclang and scsynth under **Debian**. It does not include the IDE nor the Qt GUI
+components for sclang. For compiling with GUI support, you may be able to adapt the instructions from the
+Raspberry Pi readme.
+
+Build Requirements
+------------------
+
+* BeagleBone Black (or any of the variants green, blue, industrial, pocket, etc.)
+* SD card with Debian Stretch or Buster. You could use [iot](http://beagleboard.org/latest-images),
+   [lxqt](http://beagleboard.org/latest-images) or
+   [console](https://elinux.org/Beagleboard:BeagleBoneBlack_Debian#Debian_Buster_Console_Snapshot). The console
+   version is minimal and recommended.
+* Router with ethernet internet connection for the BBB
+* Laptop connected to same network as the BBB
+* USB soundcard with headphones or speakers connected
+
+Building
+--------
+
+### Step 1: Hardware setup
+
+1. Connect an ethernet cable from the network router to the RPi
+2. Insert the SD card and USB soundcard.
+3. Connect USB power from a 5V@1A power supply
+
+### Step 2: Update the system, install required libraries
+
+ssh from your laptop by opening a terminal and typing:
+
+    ssh debian@beaglebone # default password is temppwd
+    sudo passwd debian # change password
+    sudo /opt/scripts/tools/grow_partition.sh # expand file system. skip if you are running on the emmc
+    sudo reboot # and log in again with ssh
+
+    # log in again with ssh
+    ssh debian@beaglebone
+    sudo apt-get update
+    sudo apt-get upgrade
+    sudo apt-get dist-upgrade
+    sudo apt-get install libsamplerate0-dev libsndfile1-dev libasound2-dev libavahi-client-dev libreadline-dev \
+        libfftw3-dev libudev-dev cmake git build-essential python-dev alsa-utils cpufrequtils
+
+### Step 3: Compile, install and configure jackd (no d-bus)
+
+    cd ~ # or cd into the directory where you'd like to build jack2
+    git clone git://github.com/jackaudio/jack2 --depth 1
+    cd jack2
+    ./waf configure --alsa --libdir=/usr/lib/arm-linux-gnueabihf/
+    ./waf build
+    sudo ./waf install
+    sudo ldconfig
+    cd ..
+    rm -rf jack2
+    sudo sh -c "echo @audio - memlock 256000 >> /etc/security/limits.conf"
+    sudo sh -c "echo @audio - rtprio 75 >> /etc/security/limits.conf"
+
+    # `-dhw:0` is the internal soundcard. Use `-dhw:1` for USB soundcards. `aplay -l` will list available devices.
+    # Use `nano ~/.jackdrc` to edit jack settings.
+    echo /usr/local/bin/jackd -P75 -p16 -dalsa -dhw:0 -r44100 -p1024 -n3 > ~/.jackdrc
```

Comment:
there is no internal sound on bbb.
also must use sudo for aplay on bbb and `-dhw:1` should be the default.
compare with https://supercollider.github.io/development/building-beagleboneblack
subtle but important differences from rpi so please take care when copy and paste.

```
# `-dhw:1` is normally the USB soundcard. `sudo aplay -l` will list available devices.
# Use `nano ~/.jackdrc` to edit jack settings.
echo /usr/local/bin/jackd -P75 -p16 -dalsa -dhw:1 -r44100 -p1024 -n3 > ~/.jackdrc
```